spec/compiler/normalize/op_assign_spec.cr

```diff
@@ -1,8 +1,10 @@
 require "../../spec_helper"
 
 describe "Normalize: op assign" do
-  it "normalizes var +=" do
-    assert_normalize "a = 1; a += 2", "a = 1\na = a + 2"
+  ["+", "-", "*", "&+", "&-", "&*"].each do |op|
+    it "normalizes var +=" do
```

Comment:
Might be a good opportunity to update the description to `"normalizes var #{op}="`